pgmuvi/lightcurve.py

```diff
@@ -330,7 +330,29 @@ def fit(self, model = None, likelihood = None, num_mixtures = 4,
 
         #Now we're going
 
-        
+    def plot_psd(self, means, freq, scales, weights):
+        #Computing the psd for frequencies f
+        psd = compute_psd(means, freq, scales, weights)
+
+        # Initialize plot
+        fig, ax = plt.subplots(1, 1, figsize=(8, 6))
+
+        #plotting psd
+        ax.plot(freq, psd)
+        plt.show()
```

Comment:
If you accept the suggestion above for the `show` keyword, you can also change this bit:

``` suggestion
        if show:
            plt.show()
        else:
            return fig, ax
```